src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -157,28 +140,28 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     [OuterLoop]
     public static void TimeoutTest_SendTimeout5Seconds()
     {
```

Comment:
Nit: I've always wanted this test method to be retitled, because it doesn't describe what it does.  It should be something like Long_Running_Operation_Throws_TimeoutException.  Not required for this PR, but let's gradually make the method names describe what they do.